packages/offix-scheduler/src/utils/DefaultOffixExecutor.ts

```diff
@@ -0,0 +1,7 @@
+import { OffixExecutor } from "../Offix";
+
+export class DefaultOffixExecutor implements OffixExecutor {
```

Comment:
Name can be confusing - Default might mean that this is the one I should be using, but in fact this one is doing only console log. Maybe move this to unit tests or.. Rename to `ConsoleExecutor`